### PR TITLE
Final version of the project

### PR DESCRIPTION
**Pull request by asharanjith**
## What does this PR do?
This PR is raised to 

## Description of the task to be completed
- [x] Design the web page as per the wireframe and requirements
- [x] Followed Gitflow
- [x] Deploy the page using the gh-pages

### CODE REVIEW

**Review by christianbiring1 — COMMENTED** [review]
Hi @team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request
Invalid

You have submitted a project, that was not presented be all of you each one of you in their separate video (zoom preferably),
- [ ] Kindly consider presenting the video of `35' max` where each feature and asked question in the presentation outline is answered correctly :100: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by asharanjith** [comment]
@christianbiring1 The video presentation can be done tomorrow only as the questions will be mailed on Day 5 of the project.  And that is for the final assessment on Day 5. Please clarify this.

File: src/modules/movieLikes.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      }
+      return '';
+    });
+  } catch (e) {
+    return e.message;
+  }
+  return '';
+};
+
+export const addLike = async (id, likes) => {
```
Comment (rloterh) [inline src/modules/movieLikes.js:21]: - [ ]  OPTIONAL
I think you should improve upon your _likes_ feature this way:
    - Like icon should toggle to indicate _like_ or _unlike_ when a particular user clicks on the like icon.
    - Total count of likes per user should be 1

File: src/Test/commentCounter.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import commentCounterFunction from '../modules/commentCounter.js';
+
+describe('Comment Counter', () => {
```
Comment (rloterh) [inline src/Test/commentCounter.test.js:7]: - [ ] I think your provided edge cases in testing the _counter_ functions are very limited. I'll urge you to do some more search and come up with all the possible edge cases you can think about 💪😃

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Trust & Asha, 

You did extremely great on this project! 👏 👏
There are some issues that you still need to work on to prepare your project for the assessment but you are almost there 🚦 🚦

## Suggested changes
_Kindly check my inline comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you take them into account as they can make your code better._ 😃 

**You can also consider:**
- Generally, the subheading text _Author1_ and _Author2_ should be replaced with your respective full names in the ReadMe.
- Kindly endeavor to remove the guide template instructions below the MIT license.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/modules/getMovies.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+      const list = movie.genres.toString();
+      genre.innerHTML = `Genre: ${list}`;
+      genreLang.appendChild(genre);
+      const lang = document.createElement('p');
+      lang.innerHTML = `Language: ${movie.language}`;
+      genreLang.appendChild(lang);
+      movieDescription.appendChild(genreLang);
+      movieCard.appendChild(movieDescription);
+      const commentBtn = document.createElement('button');
+      commentBtn.className = 'commentBtn';
```
Comment (aimalamiri) [inline src/modules/getMovies.js:95]: [Optional] You impressed me with your wonderful design 👌but don't you think it'll make your design look more professional and beautiful if you change the button text color to white? Because it's dark text on a dark background which seems not to be a good accessibility practice.

image

**Review by aimalamiri — APPROVED** [review]
Hi @asharanjith and @Trust-007, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ApprovedDavieGIF


## Highlights

- The application is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitFlow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
